tests/coreclr/p0-net6.0-ubuntu/Makefile

```diff
@@ -26,17 +26,25 @@ appdir:
 ext2fs: appdir
 	$(MYST) mkext2 appdir ext2fs
 
+package.pem:
+	openssl genrsa -out package.pem -3 3072
+
+build-package: package.pem
+	$(MYST) fssig --roothash ext2fs > roothash
+	$(MYST) package-sgx --roothash=roothash package.pem config_256m.json
+
 run:
 	rm -rf PASSED FAILED-*
 ifndef MYST_SKIP_PR_TEST
+	$(TEST_RUNNER) $(PACKAGE_PATH) null null 30 package pr0-256m
 	$(TEST_RUNNER) $(MYST_EXEC) config_1g.json 60 ext2 pr0-1g
-	$(TEST_RUNNER) $(MYST_EXEC) config_4g.json 60 ext2 pr0-large-memory
+	$(TEST_RUNNER) $(MYST_EXEC) config_4g.json 40 ext2 pr0-large-memory 1
 	$(TEST_RUNNER) $(MYST_EXEC) config_1g.json 300 ext2 pr0-long-time
 endif
-	$(STAT) pr0-tests-all 98
+	$(STAT) pr0-tests-all 97
```

Comment:
Just curious, why the pass percentage dropped?

Reply:
I'm not sure if we ever calculated the pass percentage on CFL before. Copied the number from the first successful CFL run I had.